Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/UserInterface.java

```diff
@@ -0,0 +1,24 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+public interface UserInterface {
+
+	public String playerName();
```

Comment:
This function actually returns the name of the player. Eg. In the beginning of the game there's a popup menu that lets you select players and such. This is it. It doesn't reallly care for what is a player object. 

Though you're right it could use a better name.